### ISSUE TITLE
Unexpected token 'U', "Unexpected"... is not valid JSON

### ISSUE THREAD

**Issue by robingulla** [stackblitz/bolt.new#1527]
### Describe the bug

Importing Stackblitz Project

sb1-eedynm
Unexpected token 'U', "Unexpected"... is not valid JSON

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-eedynm

### Steps to reproduce

Click on previously created project ( Network monitor app) , 2) import error is displayed as attached 

### Expected behavior

Should fetch the previously written code 

### Screen Recording / Screenshot

![Error Bolt](https://github.com/user-attachments/assets/2b4dcf8f-8d6e-4910-bc4e-2de505027303)

### Platform

```
Browser name = Chrome
Full version = 130.0.0.0
Major version = 130
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36 Edg/130.0.0.0
performance.memory = {
  "totalJSHeapSize": 69305845,
  "usedJSHeapSize": 60818221,
  "jsHeapSizeLimit": 2160066560
}
Username = robingulla
Chat ID = 846ab95a3385
```

### Additional context

Error:
```
An unknown error occured while running an action.
```

**Comment by engelmatt** [comment]
Same on my end. It's for all of my projects. Some minutes ago the issue started to happen on both the official bolt.new page as well as on the preview page.

**Comment by kc0tlh** [comment]
@robingulla Thanks for the report and apologies for the frustrating experience! The fix is explained in #1471
